ffi/diplomat/cpp/examples/pluralrules/test.cpp

```diff
@@ -6,12 +6,10 @@
 
 #include <iostream>
 
-const std::string_view path = "../../../../../provider/testdata/data/json/";
-
 int main() {
     ICU4XLocale locale = ICU4XLocale::create("ar").value();
     std::cout << "Running test for locale " << locale.tostring().ok().value() << std::endl;
-    ICU4XDataProvider dp = ICU4XDataProvider::create_fs(path).provider.value();
+    ICU4XDataProvider dp = ICU4XDataProvider::create_test().provider.value();
```

Comment:
issue: i kinda feel like we should have a couple fs tests too? it was a bit of a deliberate choice to do both cc @sffc

Reply:
I guess in tests it's always guaranteed that we're in the ICU repo. Otherwise this won't work because the testdata won't be there.

Reply:
Yep!